contracts/oracles/BinanceOracle.sol

```diff
@@ -50,10 +59,7 @@ contract BinanceOracle is AccessControlledV8, OracleInterface {
      * @param _sidRegistryAddress Address of SID registry
      * @param _accessControlManager Address of the access control manager contract
      */
-    function initialize(
-        address _sidRegistryAddress,
-        address _accessControlManager
-    ) external initializer notNullAddress(_sidRegistryAddress) {
+    function initialize(address _sidRegistryAddress, address _accessControlManager) external initializer {
```

Comment:
We have two attributes: `sidRegistryAddress` and `feedRegistryAddress`. Only one of them can be non-zero. Would it make sense to add a new parameter in the `initialize` function to allow us to set the `feedRegistryAddress` in that function?

Moreover, we could check that one and only one of the two attributes are set to non-zero

Reply:
Let's keep it as it is to avoid changes in the `initialize` function, that would force us to keep two different codebase